apps/docs/components/Extensions.tsx

```diff
@@ -98,8 +98,8 @@ export default function Extensions() {
                   passHref
                   href={`${
                     extension.link
-                      ? `/guides/database/extensions/${extension.name}`
-                      : '/guides/database/extensions#full-list-of-extensions'
```

Comment:
Only external links will have this property. If it exists, use that. Otherwise, use a relative link to our docs using the extension's name.

Reply:
Thoughts on making `link` required and including both internal and external URLs in `extensions.json`? Feels like we could benefit from the simplicity of less moving parts (and less opportunities to accidentally link to a broken URL).

Worth noting we will likely share `extensions.json` with the dashboard to do something similar (after this PR is merged):
#15431

Internal links could be relative:
```json
{
  "name": "pg_cron",
  "comment": "Job scheduler for PostgreSQL",
  "link": "/guides/database/extensions/pg_cron",
  "tags": ["Utility"]
}
```

Reply:
Makes sense to me to be more explicit! 

I'll make that change tonight when I resolve the merge conflicts.